program/index.md

```diff
@@ -97,7 +97,7 @@ show_sidebar: true
 			8:30
 		</td>
 		<td>
-			Keynote: Idit Keidar, Technion - Israel Institute of Technology
+			Keynote: <a href="index.html#keynote1">Concurrent Data Sketches</a>, Idit Keidar, Technion - Israel Institute of Technology
```

Comment:
Minor nitpick: `href="#keynote1"` would have the same effect here.

Reply:
done